GPG.md

```diff
@@ -75,3 +76,11 @@ You can choose to sign your outgoing messages, encrypt your outgoing messages, o
 In the case of encryption the recipient's email will be used to determine the key to use.
 
 To enable this just press `g` when prompted and then make your selection.
+
+Configuration
+-------------
+
+You can set the config value `gpg.mode` to eather always or auto(recommended).
```

Comment:
Probably worth mentioning this, briefly in `API.md` too in the "global variables" section.